presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestAddIntermediateAggregations.java

```diff
@@ -56,15 +56,18 @@ public void testBasic()
                 .setSystemProperty(ENABLE_INTERMEDIATE_AGGREGATIONS, "true")
                 .setSystemProperty(TASK_CONCURRENCY, "4")
                 .on(p -> p.aggregation(af -> {
+                    p.variable("a", BIGINT);
+                    p.variable("b", BIGINT);
+                    p.variable("c", BIGINT);
                     af.globalGrouping()
                             .step(AggregationNode.Step.FINAL)
-                            .addAggregation(p.variable(p.symbol("c")), expression("count(b)"), ImmutableList.of(BIGINT))
+                            .addAggregation(p.variable(p.symbol("c")), p.rowExpression("count(b)"))
```

Comment:
These changes should go with the commit that adds `rowExpression`.

Reply:
We cannot, that commit only added support for `rowExpression` but the rule `AddIntermediateAggregations` still cannot handle `RowExpression`. 
If we move earlier, this test will fail.

Reply:
Then merge them all together 😂 Unless the diff would become too large, in general we should introduce function in the same commit they are used. But up to you in this case though. No strong opinions.

Reply:
Then merge them all together 😂 Unless the diff would become too large, in general we should introduce function in the same commit they are used. But up to you in this case. No strong opinions.